docs/permissions/getting-started.md

```diff
@@ -20,20 +20,12 @@ The permissions framework depends on a few other Backstage systems, which must b
 
 The permissions framework itself is new to Backstage and still evolving quickly. To ensure your version of Backstage has all the latest permission-related functionality, it’s important to upgrade to the latest version. The [Backstage upgrade helper](https://backstage.github.io/upgrade-helper/) is a great tool to help ensure that you’ve made all the necessary changes during the upgrade!
 
-### Enable service-to-service authentication
-
-Service-to-service authentication allows Backstage backend code to verify that a given request originates from elsewhere in the Backstage backend. This is useful for tasks like collation of catalog entities in the search index. This type of request shouldn’t be permissioned, so it’s important to configure this feature before trying to use the permissions framework.
-
-To set up service-to-service authentication, follow the [service-to-service authentication docs](../auth/service-to-service-auth.md).
-
 ### Supply an identity resolver to populate group membership on sign in
 
 **Note**: If you are working off of an existing Backstage instance, you likely already have some form of an identity resolver set up.
 
 Like many other parts of Backstage, the permissions framework relies on information about group membership. This simplifies authoring policies through the use of groups, rather than requiring each user to be listed in the configuration. Group membership is also often useful for conditional permissions, for example allowing permissions to act on an entity to be granted when a user is a member of a group that owns that entity.
 
-[The IdentityResolver docs](../auth/identity-resolver.md) describe the process for resolving group membership on sign in.
-
```

Comment:
Should we remove this, or is there a new version we can point to? Seems odd to point to the old backend for this part :thinking:

Reply:
This has been updated just recently! Check it out! So you can put back this text